src/services/utilServices/SitesService.js

```diff
@@ -29,12 +29,38 @@ const ISOMER_ADMIN_REPOS = [
 ]
 
 class SitesService {
-  constructor({ gitHubService, configYmlService }) {
+  constructor({
+    gitHubService,
+    configYmlService,
+    usersService,
+    isomerAdminsService,
+  }) {
     this.githubService = gitHubService
     this.configYmlService = configYmlService
+    this.usersService = usersService
+    this.isomerAdminsService = isomerAdminsService
+  }
+
+  async getEmailUserSites(userId) {
+    const user = await this.usersService.findSitesByUserId(userId)
+    if (user) {
+      const { site_members: siteMemberEntries } = user
+      return siteMemberEntries.map((entry) => {
+        const repoData = entry.repo
+        return repoData.name
+      })
+    }
+    return []
```

Comment:
we could flip the condition so this reads more smoothly

Reply:
https://github.com/isomerpages/isomercms-backend/pull/480/commits/9607bdc51f19e84a9c33cb909573417825a46ad8